### PR TITLE
Issue #DP-41 Task: Option to skip review by contributing org if it is sourcing

### PR DESCRIPTION
**Pull request by niteshkesarkar**

### CODE REVIEW

File: src/app/client/src/app/modules/program/components/create-program/create-program.component.ts [hunk 1]
```diff
@@ -620,6 +621,7 @@ export class CreateProgramComponent implements OnInit, AfterViewInit {
         // tslint:disable-next-line:max-line-length
         _.find(_.find(this.programConfig.components, { id: 'ng.sunbird.collection' }).config.filters.implicit, { code: 'framework' }).defaultValue = this.userFramework;
       }
+      this.programConfig.skip_two_level_review = this.createProgramForm.value.skip_two_level_review;
       this.programData['sourcing_org_name'] = this.userprofile.rootOrgName;
       this.programData['rootorg_id'] = this.userprofile.rootOrgId;
       this.programData['createdby'] = this.userprofile.id;
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/program/components/create-program/create-program.component.ts:627]: delete that skip_two_level_review from programData

File: src/app/client/src/app/modules/program/components/create-program/programconfig.ts [hunk 2]
```diff
@@ -1,5 +1,6 @@
 // tslint:disable:max-line-length
 var programConfig = {
+    'skip_two_level_review' : false,
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/program/components/create-program/programconfig.ts:3]: can this be other way around.. two_level_review, yes or no

File: src/app/client/src/app/modules/program/components/create-program/create-program.component.ts (hunk 1 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/program/components/create-program/create-program.component.ts:627]: ok

File: src/app/client/src/app/modules/program/components/create-program/programconfig.ts (hunk 2 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/program/components/create-program/programconfig.ts]: yes, updated accordingly.

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts [hunk 3]
```diff
@@ -172,6 +172,12 @@ export class ChapterListComponent implements OnInit, OnChanges, OnDestroy, After
     } else {
       if (!_.isEmpty(this.collectionHierarchy)) { this.lastOpenedUnit(this.collectionHierarchy[0].identifier)}
     }
+    if (_.get(this.programContext, 'config.two_level_review') === 'No'
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts:175]: two_level_review should be true or false.

File: src/app/client/src/app/modules/program/components/create-program/programconfig.ts [hunk 4]
```diff
@@ -1,5 +1,6 @@
 // tslint:disable:max-line-length
 var programConfig = {
+    'two_level_review' : 'Yes',
```
Comment (twsvaishali) [inline src/app/client/src/app/modules/program/components/create-program/programconfig.ts:3]: two_level_review should be true bydefault.

Comment (niteshkesarkar) [reply src/app/client/src/app/modules/program/components/create-program/programconfig.ts]: updated

File: src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts (hunk 3 above)
Comment (niteshkesarkar) [reply src/app/client/src/app/modules/cbse-program/components/chapter-list/chapter-list.component.ts]: updated

**Pull request merged**